.github/ISSUE_TEMPLATE/support.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Support Request
+about: Support request or question relating to Kubernetes federation v2
+
+---
+
+<!-- 
+STOP -- PLEASE READ!
+
+GitHub is not the right place for support requests.
+
+If you're looking for help, check [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes) and the [troubleshooting guide](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/).
+
+You can also post your question on the [Kubernetes Slack](http://slack.k8s.io/) or the [Discuss Kubernetes](https://discuss.kubernetes.io/) forum.
+
+If the matter is security related, please disclose it privately via https://kubernetes.io/security/.
```

Comment:
This comment can likely be removed, since there is no channel for federation-specific security issues at this time.